docs/java-rest/high-level/aggs-builders.asciidoc

```diff
@@ -59,21 +59,21 @@ This page lists all the available aggregations with their corresponding `Aggrega
+| {ref}/search-aggregations-pipeline-inference-bucket-aggregation.html[Inference]                    | {javadoc-client}/analytics/InferencePipelineAggregationBuilder.html[InferencePipelineAggregationBuilder]                        | None
+| {ref}/search-aggregations-pipeline-max-bucket-aggregation.html[Max Bucket]                         | {agg-ref}/pipeline/MaxBucketPipelineAggregationBuilder.html[MaxBucketPipelineAggregationBuilder]                                | {agg-ref}/PipelineAggregatorBuilders.html#maxBucket-java.lang.String-java.lang.String-[PipelineAggregatorBuilders.maxBucket()]
+| {ref}/search-aggregations-pipeline-min-bucket-aggregation.html[Min Bucket]                         | {agg-ref}/pipeline/MinBucketPipelineAggregationBuilder.html[MinBucketPipelineAggregationBuilder]                                | {agg-ref}/PipelineAggregatorBuilders.html#minBucket-java.lang.String-java.lang.String-[PipelineAggregatorBuilders.minBucket()]
+| {ref}/search-aggregations-pipeline-sum-bucket-aggregation.html[Sum Bucket]                         | {agg-ref}/pipeline/SumBucketPipelineAggregationBuilder.html[SumBucketPipelineAggregationBuilder]                                | {agg-ref}/PipelineAggregatorBuilders.html#sumBucket-java.lang.String-java.lang.String-[PipelineAggregatorBuilders.sumBucket()]
+| {ref}/search-aggregations-pipeline-stats-bucket-aggregation.html[Stats Bucket]                     | {agg-ref}/pipeline/StatsBucketPipelineAggregationBuilder.html[StatsBucketPipelineAggregationBuilder]                            | {agg-ref}/PipelineAggregatorBuilders.html#statsBucket-java.lang.String-java.lang.String-[PipelineAggregatorBuilders.statsBucket()]
+| {ref}/search-aggregations-pipeline-extended-stats-bucket-aggregation.html[Extended Stats Bucket]   | {agg-ref}/pipeline/ExtendedStatsBucketPipelineAggregationBuilder.html[ExtendedStatsBucketPipelineAggregationBuilder]            | {agg-ref}/PipelineAggregatorBuilders.html#extendedStatsBucket-java.lang.String-java.lang.String-[PipelineAggregatorBuilders.extendedStatsBucket()]
+| {ref}/search-aggregations-pipeline-percentiles-bucket-aggregation.html[Percentiles Bucket]         | {agg-ref}/pipeline/PercentilesBucketPipelineAggregationBuilder.html[PercentilesBucketPipelineAggregationBuilder]                | {agg-ref}/PipelineAggregatorBuilders.html#percentilesBucket-java.lang.String-java.lang.String-[PipelineAggregatorBuilders.percentilesBucket()]
+| {ref}/search-aggregations-pipeline-movavg-aggregation.html[Moving Average]                         | {agg-ref}/pipeline/MovAvgPipelineAggregationBuilder.html[MovAvgPipelineAggregationBuilder]                                      | {agg-ref}/PipelineAggregatorBuilders.html#movingAvg-java.lang.String-java.lang.String-[PipelineAggregatorBuilders.movingAvg()]
```

Comment:
Thi link of `MovAvgPipelineAggregationBuilder` is broken. I cannot find the appropriate aggregation in the package summary: https://snapshots.elastic.co/javadoc/org/elasticsearch/elasticsearch/8.0.0-SNAPSHOT/org/elasticsearch/search/aggregations/pipeline/package-summary.html